### PR TITLE
[USING] Add  "how to set and enable Lock Screen"

### PR DESCRIPTION
**Pull request by davterra**
I noticed in the FAQ there is guidance as to how to delete a forgotten PIN for the Lock Screen feature, but there doesnt seem to be instructions anywhere detailing how to enable/activate/disable the Lock Screen.

So I put this together. I wouldnt have thought to put it in the "advanced" section, but that's where "Lurking Wife Mode" instructions are, and that function seems even more basic than this.

It looks like the basic "Using Wasabi" section is being reserved for the core functions....Send, Receive, etc.

### CODE REVIEW

File: docs/using-wasabi/LockScreen.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+{
+  "title": "Lock Screen",
+  "description": "A feature that allows you to lock your Wasabi wallet from being viewed or used while it is running, and then unlock it at any time with a user-specified PIN. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/LockScreen.md]:
```suggestion
  "description": "A feature that allows you to lock your Wasabi Wallet from being viewed or used while it is running, and then unlock it at any time with a user-specified PIN. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
```

File: docs/using-wasabi/LockScreen.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+}
+---
+
+# Lock Screen
+
+The `Lock Screen` feature allows you to lock your Wasabi wallet from being viewed or used while it is running, and then unlock it at any time with a user-specified PIN.
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/LockScreen.md]:
```suggestion
The `Lock Screen` feature allows you to lock your Wasabi Wallet from being viewed or used while it is running, and then unlock it at any time with a user-specified PIN.
```

File: docs/using-wasabi/LockScreen.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+[[toc]]
+
+## How to set the Lock Screen feature
+
+You can active the `Lock Screen` feature by going to the menu bar on the top/left corner of your Wasabi wallet and clicking on `Tools`.
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/LockScreen.md]:
```suggestion
You can active the `Lock Screen` feature by going to the menu bar on the top/left corner of your Wasabi Wallet and clicking on `Tools`.
```

File: docs/using-wasabi/LockScreen.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+![](/LockScreenSet2.png)
+
+Choose and enter a numerical PIN up to 10 digits in length in the PIN box, and click on `Set`.
+
+## How to lock your screen
+
+You can now activate the Lock Screen feature in two ways:
+
+1. Hit `Ctrl-l` on your keyboard.
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/LockScreen.md]:
```suggestion
1. Hit `Ctrl-L` on your keyboard.
```

File: docs/using-wasabi/LockScreen.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+1. Go to the top/left menu bar and click on `Tools`, then `Settings`.
+2. Scroll down the `Settings` page to the Lock Screen section.
+3. Enter your PIN in the PIN box, and click `Clear`.
+4. If a new PIN is desired, enter it in the PIN box and click `Set`, otherwise the `Lock Screen` feature is now disabled.
+
+## If you forget your PIN
+
+If you have forgotten your PIN, you can delete it with these steps:
+
+1. Shut down your Wasabi wallet.
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/LockScreen.md]:
```suggestion
1. Shut down Wasabi Wallet.
```

File: docs/using-wasabi/LockScreen.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+2. Open your UIConfig.json file using a text editor.
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/LockScreen.md]:
```suggestion
2. Open your `UIConfig.json` file using a text editor.
```

File: docs/using-wasabi/LockScreen.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+This file can be found here:
+
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/LockScreen.md]:
```suggestion
```

**Comment by davterra** [comment]
Thanks for those changes @riccardomasutti
I will commit them all when I get home later.

My smartphone is not quite smart enough to allow me to do that.

**Comment by RiccardoMasutti** [comment]
@davterra Click on 'Desktop Version' and enjoy the magic

**Comment by davterra** [comment]
Yeah, strangely that's not on the menu on this browser. I could try another browser, but I'm at the gym, and it can wait.

**Comment by davterra** [comment]
Brave mobile browser had desktop mode

**Comment by davterra** [comment]
Thanks Max. I agree and will add a warning.

I knew that github is coming out with a mobile app and tried to be a beta tester, but the list was full.

**Comment by davterra** [comment]
@MaxHillebrand I added the warning that you suggested. Let me know if you think it is too weak or too strong.

File: docs/using-wasabi/LockScreen.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+# Lock Screen
+
+The `Lock Screen` feature allows you to lock your Wasabi Wallet from being viewed or used while it is running, and then unlock it at any time with a user-specified PIN.
+
+[[toc]]
+
+:::warning Caution!
+This feature is only intended to be used if you are stepping away from your computer for a very short time.
+
+The PIN can be deleted and your wallet accessed by anyone that gets physical access to your computer and knows how to find and alter your UIConfig.json file.
```
Comment (yahiheb) [inline docs/using-wasabi/LockScreen.md]:
```suggestion
The PIN can be deleted and your wallet accessed by anyone that gets physical access to your computer and knows how to find and alter your `UIConfig.json` file.
```

File: docs/using-wasabi/LockScreen.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+
+Then, scroll down the `Settings` page until you see the `Lock Screen` section.
+
+![](/LockScreenSet2.png)
+
+Choose and enter a numerical PIN up to 10 digits in length in the PIN box, and click on `Set`.
+
+## How to lock your screen
+
+You can now activate the Lock Screen feature in two ways:
```
Comment (yahiheb) [inline docs/using-wasabi/LockScreen.md]:
```suggestion
You can now lock your screen in two ways:
```

File: docs/using-wasabi/LockScreen.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+To unlock, just enter the correct PIN in the provided box.
+
+![](/LockedScreen.png)
+
+## How to clear your Lock Screen PIN
+
+If you wish to delete or change your PIN:
+
+1. Go to the top/left menu bar and click on `Tools`, then `Settings`.
+2. Scroll down the `Settings` page to the Lock Screen section.
```
Comment (yahiheb) [inline docs/using-wasabi/LockScreen.md]:
```suggestion
2. Scroll down the `Settings` page to the `Lock Screen` section.
```

File: docs/using-wasabi/LockScreen.md [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+4. If a new PIN is desired, enter it in the PIN box and click `Set`, otherwise the `Lock Screen` feature is now disabled.
+
+## If you forget your PIN
+
+If you have forgotten your PIN, you can delete it with these steps:
+
+1. Shut down Wasabi Wallet.
+2. Open your `UIConfig.json` file using a text editor.
+
+This file can be found here:
```
Comment (yahiheb) [inline docs/using-wasabi/LockScreen.md]:
```suggestion
This file can be found in [Wasabi's Data Folder](/FAQ/FAQ-UseWasabi.md#where-can-i-find-the-wasabi-data-folder).
```

File: docs/using-wasabi/LockScreen.md [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
+* Windows: `/Users/{your username}/AppData/Roaming/WalletWasabi/Client`
```
Comment (yahiheb) [inline docs/using-wasabi/LockScreen.md]:
```suggestion
```

File: docs/using-wasabi/LockScreen.md [hunk 13]
```diff
@@ -0,0 +1,82 @@
...
+* Linux: `/Home/.walletwasabi/client`
```
Comment (yahiheb) [inline docs/using-wasabi/LockScreen.md]:
```suggestion
```

File: docs/using-wasabi/LockScreen.md [hunk 14]
```diff
@@ -0,0 +1,82 @@
...
+* MacOS: `/Users/{your username}/.walletwasabi/client`
```
Comment (yahiheb) [inline docs/using-wasabi/LockScreen.md]:
```suggestion
```

File: docs/using-wasabi/LockScreen.md [hunk 15]
```diff
@@ -0,0 +1,82 @@
...
+
```
Comment (yahiheb) [inline docs/using-wasabi/LockScreen.md]:
```suggestion
```

File: docs/using-wasabi/LockScreen.md [hunk 16]
```diff
@@ -0,0 +1,82 @@
...
+:::tip
```
Comment (yahiheb) [inline docs/using-wasabi/LockScreen.md]:
```suggestion
```

File: docs/using-wasabi/LockScreen.md [hunk 17]
```diff
@@ -0,0 +1,82 @@
...
+You need to mark the “show hidden files” setting to see it.
```
Comment (yahiheb) [inline docs/using-wasabi/LockScreen.md]:
```suggestion
```

File: docs/using-wasabi/LockScreen.md [hunk 18]
```diff
@@ -0,0 +1,82 @@
...
+:::
```
Comment (yahiheb) [inline docs/using-wasabi/LockScreen.md]:
```suggestion
```

File: docs/using-wasabi/LockScreen.md [hunk 19]
```diff
@@ -0,0 +1,82 @@
...
+3. Enter your PIN in the PIN box, and click `Clear`.
+4. If a new PIN is desired, enter it in the PIN box and click `Set`, otherwise the `Lock Screen` feature is now disabled.
+
+## If you forget your PIN
+
+If you have forgotten your PIN, you can delete it with these steps:
+
+1. Shut down Wasabi Wallet.
+2. Open your `UIConfig.json` file using a text editor.
+
```
Comment (yahiheb) [inline docs/using-wasabi/LockScreen.md]:
```suggestion
```

File: docs/using-wasabi/LockScreen.md [hunk 20]
```diff
@@ -0,0 +1,82 @@
...
+This file can be found here:
+* Windows: `/Users/{your username}/AppData/Roaming/WalletWasabi/Client`
+* Linux: `/Home/.walletwasabi/client`
+* MacOS: `/Users/{your username}/.walletwasabi/client`
+
+:::tip
+You need to mark the “show hidden files” setting to see it.
+:::
+
+3. Find the line at the bottom of this file called `LockScreenPinHash`, and delete the string of characters that are between the two quotation marks.
```
Comment (yahiheb) [inline docs/using-wasabi/LockScreen.md]:
```suggestion
3. Find `LockScreenPinHash` at the bottom of this file, and delete the string of characters that are between the two quotation marks.
```

File: docs/using-wasabi/LockScreen.md (hunk 11 above)
Comment (davterra) [reply docs/using-wasabi/LockScreen.md]: Why send the user to a different part of the website just to get the info that is next on this page?
The info here is copy/pasted from that same FAQ page

Comment (yahiheb) [reply docs/using-wasabi/LockScreen.md]: What if we have to change anything in that copy-pasted text? then we have to do it everywhere we copy-pasted it.

Also should we then remove all the links from every page and pull all the info in one place to not send the user to a different part?

If a reader knows what `UIConfig.json` file is and where to find it then they don't have to read that part, but if they don't they can open that link and see what it is.

Comment (davterra) [reply docs/using-wasabi/LockScreen.md]: Ok....I can go with that.
Thank you.

Comment (yahiheb) [reply docs/using-wasabi/LockScreen.md]: Thank you.

File: docs/using-wasabi/LockScreen.md [hunk 21]
```diff
@@ -0,0 +1,73 @@
...
+
+Do not treat this feature as a strong, fail-safe lock on your wallet.
+:::
+
+## How to set the Lock Screen feature
+
+You can active the `Lock Screen` feature by going to the menu bar on the top/left corner of your Wasabi Wallet and clicking on `Tools`.
+In the resulting dropdown menu, click on `Settings`.
+
+![](/LockScreenSet1.png)
```
Comment (MaxHillebrand) [inline docs/using-wasabi/LockScreen.md]:
```suggestion
```
Do we really need this screenshot? Nowhere else in the docs do we have a screenshot for it.

**Pull request merged**